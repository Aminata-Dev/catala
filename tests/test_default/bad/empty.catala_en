## Article

```catala
declaration scope A:
  context x content integer
  context y content boolean

scope A:
  definition x equals 1
```

```catala-test-inline
$ catala Interpret -s A
[WARNING] In scope "A", the variable "x" is never used anywhere; maybe it's unnecessary?

┌─⯈ tests/test_default/bad/empty.catala_en:5.10-11:
└─┐
5 │   context x content integer
  │           ‾
  └─ Article
[WARNING] In scope "A", the variable "y" is never used anywhere; maybe it's unnecessary?

┌─⯈ tests/test_default/bad/empty.catala_en:6.10-11:
└─┐
6 │   context y content boolean
  │           ‾
  └─ Article
[ERROR] This variable evaluated to an empty term (no rule that defined it applied in this situation)

┌─⯈ tests/test_default/bad/empty.catala_en:6.10-11:
└─┐
6 │   context y content boolean
  │           ‾
  └─ Article
#return code 255#
```
