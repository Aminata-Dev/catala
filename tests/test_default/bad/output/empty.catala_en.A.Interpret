[ERROR] This variable evaluated to an empty term (no rule that defined it applied in this situation)

 --> tests/test_default/bad/empty.catala_en
  | 
6 |   context y content boolean
  |           ^
  + Article
