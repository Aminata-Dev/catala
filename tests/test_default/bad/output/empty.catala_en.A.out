[ERROR] This variable evaluated to an empty term (no rule that defined it applied in this situation)

 --> test_default/bad/empty.catala_en
  | 
6 |   context y content boolean
  |           ^
  + Article
